src/torznab.ts

```diff
@@ -631,6 +632,14 @@ export async function validateTorznabUrls() {
 			);
 		}
 	}
+
+	const uniqueUrls = new Set(
+		urls.map((url) => url.origin + "?" + url.searchParams.toString()),
+	);
+	for (const host of uniqueUrls) {
+		await checkArrIsActive(host, "Torznab");
+	}
+
```

Comment:
Torznab servers aren't really arr instances, so I don't think we should use code from `arr.ts` for them. What does this do that fetching the caps doesn't already do?

Reply:
We discussed the reason I did this the other day...because it relies on timing out of all indexers during cap fetching otherwise.

This prevents the need for that and checks the unique torznab hosts before fetching caps. The arr code works for this purpose perfectly.

Reply:
Would you prefer a set timeout for fetchCaps over this approach?

Reply:
It looks like this just moves the timeout to an earlier call. We do the fetchCaps calls in parallel (below) so I think the 10 second timeout on caps calls is a sufficient solution here. 

 https://github.com/cross-seed/cross-seed/blob/136010fe01e9d086892dffc09d78ed03d9be6786/src/torznab.ts#L596